packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -232,13 +234,14 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
     // If we are open or we are just closed, shouldFocusAfterClose is set,
     // are focused but we are not the activeElement set focus on the input
     if (
-      isOpen ||
-      (prevState.isOpen &&
-        !isOpen &&
-        this._focusInputAfterClose &&
-        focused &&
-        this._autofill.current &&
-        document.activeElement !== this._autofill.current.inputElement)
+      this._focusInputAfterBlur &&
+      (isOpen ||
+        (prevState.isOpen &&
+          !isOpen &&
+          this._focusInputAfterClose &&
+          focused &&
+          this._autofill.current &&
```

Comment:
This if check (and below) are getting unruly, consider pulling them into their own helper functions with good comments on the different sets of conditions

Reply:
Will do, I was actually trying to figure out how to get rid of _focusInputAfterBlur and use focused instead.